src/webview/manage-repository/app/home.tsx

```diff
@@ -0,0 +1,95 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Box, Button, Container, FormControlLabel, Radio, RadioGroup, Stack, Typography } from '@mui/material';
+import { DefaultProps } from '../../common/propertyTypes';
+import './home.scss';
+import { AddRepository } from './addRepository';
+import { ShowRepositories } from './showRepositories';
+import { VSCodeMessage } from './vsCodeMessage';
+
+export class ManageRepository extends React.Component<DefaultProps, {
+    value: string
+}> {
+
+    constructor(props: DefaultProps | Readonly<DefaultProps>) {
+        super(props);
+        this.state = {
+            value: 'add'
+        };
+    }
+
+    componentDidMount(): void {
+        VSCodeMessage.onMessage((message) => {
+            if (message.data.action === 'addRepo' && message.data.status) {
+                this.setState({
+                    value: 'show'
+                });
+                VSCodeMessage.postMessage({
+                    action: `getRepositoryList`
+                });
+            }
+        });
+    }
+
+    handleOptionChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
+
+        this.setState({
+            value: event.target.value
+        });
+
+        if (event.target.value === 'show') {
+            VSCodeMessage.postMessage({
+                action: `getRepositoryList`
+            });
+        }
+    }
+
+    render(): React.ReactNode {
+        const { value } = this.state;
+        return (
+            <div className='mainContainer margin'>
+                <div className='title'>
+                    <Typography variant='h5'>Manage Repositories</Typography>
+                </div>
+                <div className='subTitle'>
+                    <Typography>Manage template repositories installed on disk at either the default location (~/.config/func/repositories) or the location specified by the --repository flag. Once added, a template from the repository can be used when creating a new function.</Typography>
```

Comment:
@msivasubramaniaan Where are we providing the option to user to specify the location of the template repository ?

Because we need to mention that option and not the --repository flag in the description.

Reply:
Sorry missed that one. will work and push the code

Reply:
I will handle the above scenario on this [issue](https://github.com/redhat-developer/vscode-openshift-tools/issues/3128)